packages/core/src/messagingProtocols/mls/E2EIdentityService/E2EIServiceExternal.ts

```diff
@@ -285,7 +285,7 @@ export class E2EIServiceExternal extends TypedEventEmitter<Events> {
   }
 
   private async addCrlDistributionTimer({expiresAt, url}: {expiresAt: number; url: string}): Promise<void> {
-    await this.coreDatabase.add('crls', {expiresAt, url}, url);
+    await this.coreDatabase.put('crls', {expiresAt, url}, url);
```

Comment:
what's the exact difference here?

Reply:
Docs says that `.add` rejects if an item of a given key already exists in the store, while `.put` puts an item in the database and replaces any item with the same key. 

It's very likely that the method with an already existing in the store key (url), so it's more safe to use put which will alternatively just update the existing entry.